package.json

```diff
@@ -26,5 +31,6 @@
     },
     "dependencies": {
         "ethers": "^6.13.2"
-    }
+    },
+    "packageManager": "yarn@1.22.21+sha1.1959a18351b811cdeedbd484a8f86c3cc3bbaf72"
```

Comment:
Why do we need yarn here over npm? If so, you need to update `ci.yml` and delete `package-lock.json`

Reply:
`yarn` is used for publishing, because if I remember correctly `npm publish` does not allow specifying the version on the CLI, which we need, because the version is set dynamically on release.

https://github.com/zeta-chain/protocol-contracts-ton/pull/22/commits/e0320e638a549aea8e074740c0e10f205e9f0c5d